src/lib/typeCommand.ts

```diff
@@ -1,5 +1,5 @@
 import { EmbedData } from 'discord.js';
-import { CommandDefinition, isMessageCommand, MessageCommandDefinition, hasRequiredPermissions, sendPermissionsEmbed } from './command';
+import { CommandDefinition, isMessageCommand, MessageCommandDefinition, hasRequiredPermissions, sendPermissionsEmbed, replyToMessage, replyToCommandOrQuestion } from './command';
```

Comment:
```suggestion
import { CommandDefinition, isMessageCommand, MessageCommandDefinition, hasRequiredPermissions, sendPermissionsEmbed, replyToCommandOrQuestion } from './command';
```
`replyToMessage` no longer exists and will cause a compile error.